### PR TITLE
 Delete serialization tests from ieFuncTests

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - Move serialization tests from ieFuncTests to ov_core_unit_tests
 - Fix clang-format

### Tickets:
 - CVS-95525

### CODE REVIEW

File: src/core/tests/serialization/single_layer/activation.cpp [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace ngraph::helpers;
+namespace {
+TEST_P(ActivationLayerTest, Serialize) {
+    Serialize();
+}
+// Common params
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
```
Comment (ilyachur) [inline src/core/tests/serialization/single_layer/activation.cpp:18]: Please use new API

File: src/core/tests/serialization/single_layer/activation.cpp [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    InferenceEngine::Precision::FP32
+    // TODO: Fix Issue-27390
+    // InferenceEngine::Precision::I16,
+    // InferenceEngine::Precision::U8
```
Comment (ilyachur) [inline src/core/tests/serialization/single_layer/activation.cpp:22]:
```suggestion
    InferenceEngine::Precision::I16,
    InferenceEngine::Precision::U8
```

**Review by ilyachur — COMMENTED** [review]
Honestly I think that our code should be covered by serializer tests and visitor API tests. If we just remove these tests, how will test coverage change?

**Comment by olpipi** [comment]
> Honestly I think that our code should be covered by serializer tests and visitor API tests. If we just remove these tests, how will test coverage change?

Let me check it

**Comment by olpipi** [comment]
Measured coverage for core and ir frontend with ov_core_unit_tests only. Coverage drops a little.
Core: with these tests - 61.6 %. wo tests - 61.6 %
ir frontend with tests - 60.8 %. wo tests - 58.6 %

@ilyachur, do we need to delete these tests?

**Comment by ilyachur** [comment]
> Measured coverage for core and ir frontend with ov_core_unit_tests only. Coverage drops a little. Core: with these tests - 61.6 %. wo tests - 61.6 % ir frontend with tests - 60.8 %. wo tests - 58.6 %
> 
> @ilyachur, do we need to delete these tests?

@olpipi Yes I think we need extend ir frontend tests and remove these serialization tests

**Comment by ilyachur** [comment]
Please analyse and move some tests to IR frontend in order to recover the test coverage (you don't need to serialize models, but looks like some parser parts were covered by these tests).

**Review by ilyachur — APPROVED** [review]

**Pull request merged**